core/src/integration-test/java/com/scalar/db/storage/cosmos/CosmosCrossPartitionScanIntegrationTest.java

```diff
@@ -27,6 +27,11 @@ protected Map<String, String> getCreationOptions() {
     return CosmosEnv.getCreationOptions();
   }
 
+  @Override
+  protected int getThreadNum() {
```

Comment:
Can we override the `isParallelDdlSupported()` method instead of this? Or we might be able to remove the `isParallelDdlSupported()` method and use `getThreadNum()` instead for the sake of consistency.

Reply:
@brfrn169 Good point! Actually I noticed the same and created a branch to remove `executeDdls()` and `isParallelDdlSupported()` https://github.com/scalar-labs/scalardb/commit/a17d246347ae2c5b79f2e5335ff311056f02a9fe. 

I'll create a PR with the change for only branches `3` and `master` that contain those unnecessary methods (reducing concurrency for Cosmos DB integration test is needed for all the release and support branches)